### DISCUSSION TITLE
Full Windows keyboard

### DISCUSSION THREAD

**Discussion by lipedantass** [AppGround-io/bluetooth-keyboard-and-mouse-support#123] [Ideas]
Is it possible to have access to full keyboard like the virtual keyboard on windows 10. With all the key(Ctrl, Shif, Alt, Tab...) even if my android default keyboard doesn't have this functionality?
![1200px-Croatian_keyboard_layout](https://user-images.githubusercontent.com/22300912/96350102-c5dd5180-10ab-11eb-9e02-c0cb0d06be08.jpg)

**Comment by zorazor** [comment]
Maybe it's an option to (re)use the shortcut keyboard... but it would probably need a lot of changes anyway.

![Screenshot_20201018-185110_Bluetooth Keyboard   Mouse](https://user-images.githubusercontent.com/54184078/96374685-5ed79f80-1174-11eb-99b8-27febcb27b7d.jpg)
